src/SFA.DAS.EmployerCommitmentsV2.Web/Startup/PersistDataStorageStartup.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.EmployerCommitmentsV2.Configuration;
+using SFA.DAS.EmployerCommitmentsV2.Web.Configurations;
+using StackExchange.Redis;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Startup
+{
+    public static class PersistDataStorageStartup
```

Comment:
I'd recommend calling this DataProtectionStartupExtensions or something similar.